File: dotNetTips.Utility/TriState.vb
' ***********************************************************************
' Assembly         : dotNetTips.Utility
' Author           : David McCarter
' Created          : 05-15-2014
'
' Last Modified By : David McCarter
' Last Modified On : 05-15-2014
' ***********************************************************************
' <copyright file="TriState.vb" company="David McCarter Consulting">
'     David McCarter Consulting. All rights reserved.
' </copyright>
' <summary></summary>
' ***********************************************************************
''' <summary>
'''
''' </summary>
Public Enum Tristate
    ''' <summary>
    ''' The false
    ''' </summary>
    ''' <remarks></remarks>
    [False] = 0
    ''' <summary>
    ''' The true
    ''' </summary>
    ''' <remarks></remarks>
    [True] = -1
    ''' <summary>
    ''' The use default
    ''' </summary>
    ''' <remarks></remarks>
    UseDefault = -2
End Enum

File: dotNetTips.Utility.Standard/TriState.cs
// ***********************************************************************
// Assembly         : dotNetTips.Utility.Standard
// Author           : David McCarter
// Created          : 01-23-2017
//
// Last Modified By : David McCarter
// Last Modified On : 01-23-2017
// ***********************************************************************
// <copyright file="TriState.cs" company="dotNetTips.Utility.Standard">
//     Copyright (c) dotNetTips.com - McCarter Consulting. All rights reserved.
// </copyright>
// <summary></summary>
// ***********************************************************************

/// <summary>
/// The dotNetTips.Utility.Standard namespace.
/// </summary>
namespace dotNetTips.Utility.Standard
{
    /// <summary>
    /// Enum TriState
    /// </summary>
    public enum Tristate
    {
        /// <summary>
        /// False
        /// </summary>
        False = 0,

        /// <summary>
        /// True
        /// </summary>
        True = -1,

        /// <summary>
        /// Use default
        /// </summary>
        UseDefault = -2
    }
}